exercises/isbn-verifier/package.yaml

```diff
@@ -0,0 +1,20 @@
+name: isbn-verifier
+version: 1.0.0.1
```

Comment:
make it 1.1.0.1

Reply:
Thats because the update ?

Reply:
indeed, because that's what https://github.com/exercism/problem-specifications/blob/master/exercises/isbn-verifier/canonical-data.json#L3 says.